src/App.js

```diff
@@ -11,6 +11,7 @@ import Mockman from "mockman-js";
 import Liked from "./pages/liked/Liked";
 import SingleVideo from "./pages/single video/SingleVideo";
 import History from "./pages/history/History";
+import WatchLater from "./pages/watch later/WatchLater";
```

Comment:
-  "./pages/watch later/WatchLater"  instead this you can add index.js in pages update it
     - example: output may look like this import { LandingPage, Cart, Wishlist, ProductWithFilter } from  './pages/index';